index.html

```diff
@@ -0,0 +1,193 @@
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+          </div>
+        </li>
+        <li class="item">
+          <img class="placeholder" src="./pics/Img-placeholder.png" alt="">
+          <div class="text-card">
+            <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
+            <ul class="courses">
+              <li>Ruby on Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+            <button type="submit">See Project</button>
+        </div>
+        </li>
+      </ul>
+    </section>
+
+    <section id="about-me">
+      <article id="about-article">
+        <h1>About me</h1>
+        <p id="about-text"> I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <button type="submit">Get My Resume</button>
+      </article>
+      <ul class="about-row">
+        <li class="about-item">
+          <img class="about-img" src="./pics/icon-languages.svg" alt="icon-languages">
+          <h3>Languages</h3>
+          <ul class="courses-about">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </li>
+        <li class="about-item">
+          <img class="about-img" src="./pics/icon-frameworks.svg" alt="icon-frameworks">
+          <h3>Frameworks</h3>
+          <ul class="courses-about">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="about-item">
+          <img class="about-img" src="./pics/icon-skills.svg" alt="icon-skills">
+          <h3>Skills</h3>
+          <ul class="courses-about">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section id="form-container">
+      <header>
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h2>
+      </header>
+      <form id="form-element" action="https://formspree.io/f/meqdoagj" method="POST">
+        <input class="input-field" type="text" name="name" maxlength="30" placeholder="Full name" required>
+        <input class="input-field" type="email" name="email" placeholder="Email address" required>
+        <textarea id="text-area" class="input-field" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+        <button id="form-button" type="submit">Get in touch</button>
+      </form>
+    </section>
```

Comment:
Kindly fix the background image of the `contact-form` section 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/187036062-e09f6e5b-0746-4b04-97b6-65fbbb1cbe63.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187036076-a6e23e78-6fe6-4cac-ad07-5c202c8bcc4e.png)